Assets/Scripts/Managers/InputManager.cs

```diff
@@ -31,18 +31,35 @@ private void HandleInput() {
 
   private void HandleLockableInput() {
     if (MouseActive) {
-      if (UIManager.Instance.UIMode == UIMode.THREE_DIMENSIONAL) {
-        Handle3DModeMouseInput();
-        Handle3DModeScrollWheelInput();
-      } else {
-        HandleTacticalModeMouseInput();
+      switch (UIManager.Instance.UIMode) {
+        case UIMode.THREE_DIMENSIONAL: {
+          Handle3DModeMouseInput();
+          Handle3DModeScrollWheelInput();
+          break;
+        }
+        case UIMode.TACTICAL: {
+          HandleTacticalModeMouseInput();
+          HandleTacticalModeScrollWheelInput();
+          break;
+        }
+        default: {
+          break;
+        }
```

Comment:
Empty default case unreachable for two-value enum, either remove or log an error or warning here